lib/showoff.rb

```diff
@@ -858,7 +858,6 @@ def update_commandline_code(slide)
           lines = out.split("\n")
           if lines.first.strip[0, 3] == '@@@'
             lang = lines.shift.gsub('@@@', '').strip
-            pre.set_attribute('class', 'highlight')
```

Comment:
The markdown renderer only generates `<pre><code>`. Showoff adds the `highlight` class.

I just tested it with the example presentation, and it properly highlights, whether using redcarpet or commonmarker. Do you have an example presentation that displays this behaviour?